tests/installation/start_install.pm

```diff
@@ -48,6 +48,7 @@ sub run {
                 while (!check_screen('all-conflicts-resolved-packages', 4)) {
                     assert_and_click 'package-conflict-choice';
                     send_key $cmd{ok};
+                    wait_still_screen 10;
```

Comment:
Let's try the following then: Next time you can provide a PR to fix fast (and ugly) and reference a WIP-PR that should improve it further :-)